streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationManageServiceImpl.java

```diff
@@ -559,6 +558,7 @@ public boolean update(Application appParam) {
                 break;
             case REMOTE:
             case YARN_SESSION:
+                application.setFlinkClusterId(appParam.getFlinkClusterId());
```

Comment:
Oh no, there's no need to make changes here.

Reply:
sorry, I had made a mistake here